src/data/accessibility_de.json

```diff
@@ -0,0 +1,55 @@
+{
+    "meta": {
+        "title": "Barrierefreiheitserklärung des Robert Koch-Instituts und Bundesministerium für Gesundheit",
+        "description": ""
+    },
+    "title": "Barrierefreiheitserklärung des Robert Koch-Instituts und Bundesministerium für Gesundheit",
+    "section-android-main": {
+        "textblock": [
+            "Diese Erklärung zur Barrierefreiheit gilt für Version 2.17 der <b>Corona-Warn-App (Android)</b> des Robert Koch-Instituts und Bundesministerium für Gesundheit.",
```

Comment:
Will this text be updated to reflect version 2.19 which will include the link to the accessibility page?

Reply:
I understand that the assessment was done with version 2.17, and not again with the following versions.

Reply:
Okay, then you maybe should think about the wording again. "Diese Erklärung zur Barrierefreiheit gilt für Version 2.17" sounds like it would be updated regularly, like the privacy policy.

Reply:
Changed wording to "Diese Erklärung zur Barrierefreiheit gilt **ab** Version 2.17" in https://github.com/corona-warn-app/cwa-website/pull/2508/commits/1801a9ce56cbb3586bc31edafb20eb9a3343cc8a, thanks for the suggestion!